src/tasks/misc.ts

```diff
@@ -127,6 +128,14 @@ export const MiscQuest: Quest = {
       limit: { tries: 1 },
       freeaction: true,
     },
+    {
```

Comment:
Single-path tasks should go under the appropriate `src/paths` directory. In this case you can put it in [CasualQuest](https://github.com/Kasekopf/loopstar/blob/main/src/paths/casual/tasks.ts#L175).